style.css

```diff
@@ -25,36 +29,113 @@ body {
 
 /* header */
 
-header {
-  position: sticky;
-  top: 0;
-  padding: 3.625rem 1.5rem 0.75rem;
-  background-color: #fff;
+.navbar {
+  min-height: 70px;
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
 }
 
-.logo {
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 40px;
+  margin-left: 820px;
+}
+
+.head {
+  font-size: 1.2em;
   font-weight: 700;
-  font-size: 1.1rem;
-  color: #6070ff;
 }
 
-header ul {
-  display: flex;
-  flex-direction: column;
-  gap: 5px;
+.nav-link {
+  transition: 0.7s ease;
+  color: black;
+}
+
+.nav-link:hover {
+  color: rgb(45, 245, 245);
+}
+
+.nav-icon {
+  display: none;
+  cursor: pointer;
 }
 
-header ul li {
-  height: 0.1rem;
+.bar {
+  display: block;
   width: 20px;
+  height: 3px;
+  margin: 5px auto;
+  margin-right: 10px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
   background-color: #6070ff;
 }
 
-header ul li a {
-  visibility: hidden;
+@media (max-width: 768px) {
+  .header {
+    width: 332px;
+  }
+
+  .nav-icon {
+    display: block;
+  }
+
+  li a {
+    color: white;
+  }
+
+  .nav-icon.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .nav-icon.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: white;
+  }
+
+  .nav-icon.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: white;
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 0;
+    gap: 0;
+    width: 100%;
+    transition: 0.3s;
+    margin-left: 0;
+    display: block;
+    padding-top: 50px;
+    margin-bottom: 30px;
+  }
+
+  .nav-link {
+    color: white;
+    font-size: 2em;
+    font-weight: 600;
+  }
+
+  .nav-item {
+    padding-bottom: 15px;
+    margin: 16px 0;
+  }
+
+  .nav-menu.active {
+    max-width: 768px;
+    height: 718px;
+    background-repeat: no-repeat;
+    left: 0;
+    background-color: #6070ffc2;
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+    margin-top: -2px;
+  }
```

Comment:
- [ ] Kindly make sure once the menu is active it covers the whole visible screen in order to meet the provided design as much as possible.
- [ ] Kindly make sure to provide the padding on the left side of the links in order to meet the Figma design

   - ### YOUR DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/216348904-47d4fb02-3352-4ff3-ab8f-f108b989186c.png)

   
   - ### FIGMA DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/216348762-a051dd44-4b70-4f4d-8214-5c1b09b1e7d3.png)